salt/base/motd.sls

```diff
@@ -0,0 +1,42 @@
+/etc/update-motd.d/10-help-text:
+  file.managed:
```

Comment:
It seems `file.managed` isn't happy about some missing values (which we may not need)

```
    salt-master: [WARNING ] State for file: /etc/update-motd.d/10-help-text - Neither 'source' nor 'contents' nor 'contents_pillar' nor 'contents_grains' was defined, yet 'replace' was set to 'True'. As there is no source to replace the file with, 'replace' has been set to 'False' to avoid reading the file unnecessarily.
```

that's probably OK, but perhaps there's a different state we can use for setting permissions?

Reply:
I think `replace` directive will fix it https://docs.saltproject.io/en/latest/ref/states/all/salt.states.file.html#salt.states.file.managed